corehq/apps/domain/forms.py

```diff
@@ -2655,3 +2655,28 @@ def clean_version(self):
         if not self.cleaned_data.get('version'):
             self.add_error('version', _("Please select version"))
         return self.cleaned_data.get('version')
+
+
+class DomainAlertForm(forms.Form):
+    text = CharField(
+        label="Text",
+        widget=forms.Textarea,
+        required=True,
+    )
```

Comment:
Keep it simple and support only text for now. 
This can be extended to add a start/end time as well which is already available on the model and is used for maintenance alerts